charts/prometheus/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: v2.53.0
-version: 25.22.1
+version: 25.22.2
```

We have to make the change minor.
```suggestion
version: 25.23.0
```